bin/runci.sh

```diff
@@ -1,13 +0,0 @@
-#!/bin/sh
-set -eux
-TEST=${1}
-
-case "$TEST" in
```

Comment:
👍 I didn't know it was possible to organize .travis.yml like that, this script was only here to get a nice overview of the build matrix on the web UI